java/com/jfreechartextensions/axistransformations/ExtendedNumberAxis.java

```diff
@@ -98,9 +98,7 @@ public void setUpperBound(final double max) {
     @Override
     public Range getRange() {
         final Range r = super.getRange();
-        if (!Double.isNaN(minRange) && !Double.isNaN(maxRange)) {
-            return new Range(minRange, maxRange);
-        } else if (!Double.isNaN(minRange)) {
+        if (!Double.isNaN(minRange)) {
```

Comment:
This change looks wrong to me.  Previously, if min and max were "normal", it would return the normal range.  Now, there is no checking of both the min and max.  Either min or max gets checked.  If this code is correct, it needs an explanatory comment.

Reply:
With the current fix for IDS-3246 in iris, we don't want [minRange, maxRange] all the times (like while zoom-in/zoom-out we wan't super.getRange() rather than [minRange, maxRange] which is managed by this change.

Also, I did test this change with other queries and they seem to work fine.